Framework/Parallel/inc/MantidParallel/IO/EventParser.h

```diff
@@ -219,7 +219,7 @@ void EventParser<TimeOffsetType>::startAsync(
   // Wrapped in lambda because std::thread is unable to specialize doParsing on
   // its own
   m_thread =
-      std::thread([this, event_id_start, event_time_offset_start, &range] {
+      std::thread([this, event_id_start, event_time_offset_start, range] {
```

Comment:
@SimonHeybrock Could I get your opinion on this change? Address sanitizer picked up a `use-after-scope` error in
[EventParserTest.h](https://github.com/mantidproject/mantid/blob/master/Framework/Parallel/test/EventParserTest.h#L229). `range` goes out of scope as the lambda survives longer than `startAsync` but range was a temporary so the reference becomes invalid.

As the range struct is small copying it into the lambda seemed the safest way to solve the issue as I don't think `startAsync` is performance critical?

Reply:
I agree with your analysis, copying 24 Bytes is fine. :+1: